### ISSUE TITLE
Issue tracker plugin

### ISSUE THREAD

**Issue by ohhmm** [RetroShare/RetroShare#499]
Hi,

Just want to share an idea.
I guess, RetroShare is ideal solution for distributed secret work on a project.
It seems to me that Jira-like plugin with bug tracking ability might be awesome add-on.

**Comment by csoler** [comment]
I totally agree. It's among our plans to make the plugin API more open and documented in the near future. I hope that will help people coding new plugins.
